### PR TITLE
Fix "With Matplotlib" and "Usage" in README.

### PR DESCRIPTION
**Pull request by kantarow**
Add an explanation for `rbenv` and `pyenv`.
`sample_images` directory was missing.

### CODE REVIEW

**Comment by youchan** [comment]
Too many branches in README! Can you rewrite to the simple description?
And I think execution `sudo` is not good by default.

File: README.md [hunk 1]
```diff
@@ -33,15 +33,29 @@ $ CONFIGURE_OPTS="--enable-shared" pyenv install 3.x.x
...
+
+If you use `pyenv` then you don't need `sudo`.
+
+```
 sudo python3 -m pip install -U pip matplotlib
 ```
 
 ## Usage
 
+You should do `mkdir sample_images` first.
```
Comment (284km) [inline README.md]: Thank you for a report.
I changed to `mkdir_p` before saving a image file, so this description is unnecessary now.

**Comment by AtAlppha** [comment]
Hi

**Review by AtAlppha — APPROVED** [review]
Add me

**Pull request closed without merging**